debug-writer/src/main/kotlin/debugwriter/Main.kt

```diff
@@ -0,0 +1,94 @@
+package debugwriter
+
+import androidx.compose.desktop.LocalAppWindow
+import androidx.compose.desktop.Window
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.IntSize
+import debugwriter.decoration.AppTheme
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.async
+
+private val fileName = "${System.getProperty("user.home")}/.debug-writer/debug-info.txt"
+private var output by mutableStateOf("")
+private var isReady by mutableStateOf(true)
```

Comment:
If we set this to 'false' by default then we will show Loading immediately on the first frame